doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,80 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+
+## v0.5.x -> v0.6.0
+
+### Breaking Change
+
+The v0.6.0 release removes [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
+If you are using operator level s3 backup in v0.5.x, you must migrate to [cluster level backup](../backup_config.md#s3-on-aws).
+
+Steps to upgrade:
+
+Assuming your cluster uses [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
+
+- Create a backup of you current data using the [backup config guide](../backup_config.md).
```

Comment:
. . . using the [backup service guide](https://github.com/coreos/etcd-operator/blob/master/doc/user/backup_service.md#http-api-v1)